pkg/cnispawn/spawn.go

```diff
@@ -45,18 +45,18 @@ func RunContainer(path string, background bool) error {
 		}
 	}
 
-	// TODO(nhlfr): Don't hardcode kubeadm-nspawn specific options,
-	// expose all the options of systemd-nspawn instead.
 	args := []string{
 		systemdNspawnPath,
 		"--capability=cap_audit_control,cap_audit_read,cap_audit_write,cap_audit_control,cap_block_suspend,cap_chown,cap_dac_override,cap_dac_read_search,cap_fowner,cap_fsetid,cap_ipc_lock,cap_ipc_owner,cap_kill,cap_lease,cap_linux_immutable,cap_mac_admin,cap_mac_override,cap_mknod,cap_net_admin,cap_net_bind_service,cap_net_broadcast,cap_net_raw,cap_setgid,cap_setfcap,cap_setpcap,cap_setuid,cap_sys_admin,cap_sys_boot,cap_sys_chroot,cap_sys_module,cap_sys_nice,cap_sys_pacct,cap_sys_ptrace,cap_sys_rawio,cap_sys_resource,cap_sys_time,cap_sys_tty_config,cap_syslog,cap_wake_alarm",
 		"--bind=/sys/fs/cgroup",
 		"--bind-ro=/boot",
 		"--bind-ro=/lib/modules",
-		"-bD",
-		path,
+		"--boot",
+		"--notify-ready=yes",
 	}
 
+	args = append(args, nspawnArgs...)
```

Comment:
@nhlfr what do you mean? Here I pass everything that you specify on the command line to nspawn